Sources/ViewControllers/RootViewController.swift

```diff
@@ -83,9 +85,18 @@ class RootViewController: UIViewController {
         }
     }
 
-    private func showScan() {
-        for child in tabBarVC.viewControllers ?? [] where child as? ScannerViewController != nil {
-            tabBarVC.selectedIndex = tabBarVC.viewControllers?.firstIndex(of: child) ?? 0
+    @objc func showScan() {
+        for child in tabBarVC.viewControllers ?? [] {
```

Comment:
Here `child` is a UINavigationController, not a ScannerViewController. One of child's children is a ScannerViewController.